specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/providers.json

```diff
@@ -158,6 +165,81 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "MetricSpecification": {
+      "description": "Metric specification of operation.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of Metric specification."
```

Comment:
Why is "Metric" capitalized here and below when it's not on line 174. It's not a proper noun and shouldn't be capitalized within a sentence.